### ISSUE TITLE
is it alive?

### ISSUE THREAD

**Issue by bam80** [tbenz9/siasync#25]
Hi, does this project still supported?

**Comment by tbenz9** [comment]
I think the short answer is "no" since I'm not actively developing it anymore.  But if someone submitted a pull request I would certainly review it and engage the developer.  Is there something you'd like to see?

**Comment by bam80** [comment]
I didn't try it yet.

Question:
If files were added while the app wasn't run, will they be synced on run?

**Comment by tbenz9** [comment]
> Question: If files were added while the app wasn't run, will they be synced on run?

Yes, when Siasync starts up it scans the source directory and syncs any files not found on Sia.

**Comment by bam80** [comment]
Thanks.
I hoped to get preliminary experience with official Sia app to get an idea if I need your app or not, but couldn't - initial sync process seems taking a days..
So could you clarify to me a key difference between adding a folder to a snapshot in official app and syncing it in your app?
I hope the orig app works that way new files added to the folder after snapshot do not make a new snapshot begin from scratch - new files will be just "added" to old snapshot and produce a new one. Does it work that way?
